keras_cv/layers/preprocessing/erasing.py

```diff
@@ -0,0 +1,196 @@
+import abc
+
+import tensorflow as tf
+import tensorflow.keras.layers as layers
+from tensorflow.python.keras.utils import layer_utils
+
+from keras_cv.utils import fill_utils
+
+
+class BaseErasing(layers.Layer, abc.ABC):
+    """This can be inherited by layers that wants to implement erasing of patches.
+
+    Args:
+        rate: Float between 0 and 1.  The fraction of samples to augment.
+        patch_value: Float or string. The value to fill in the patches. If "gaussian", will
```

Comment:
This is similar to the `fill_mode` argument in other augmentation layers. Let's call it that. Initial values should be `"gaussian_noise"` or `"constant"`. Add a `fill_value=0.0` argument for the constant mode.